episerver/content-graph-js-sdk
# CG-985: Setup OptiID and use it on musicfestival demo app

## Description
This pull request proposes several changes to the project:

* Renamed the project frontend to react-script to give it a more descriptive name.
* Cloned the react-script project and converted it to Next.js, a popular React-based framework that offers many benefits.
* Converted route-based components to route pages, which are simpler and easier to manage.
* Added support for three OIDC providers: Opti ID, EPiServer OpenIDConnect, and Azure AD, making the application more versatile.
## Motivation and Context
By making these changes, we aim to improve the quality and functionality of the project. The new name and framework will make it easier to work with, while the new routing system will simplify development. Additionally, supporting multiple OIDC providers will allow the application to be used in a variety of authentication and authorization scenarios.

## How to Test
To test this pull request, clone the repository and run the application. Verify that the new name and framework are working as expected, and that the routing system and OIDC providers are functioning correctly.

## Related Issue
This pull request addresses the following issues:

* Renaming the project.
* Converting to Next.js.
* Converting to route pages.
* Adding OIDC providers, especially Opti ID support.
## Checklist
- [x] Renamed the project to react-script
- [x] Converted to Next.js
- [x] Converted route-based components to route pages
- [x] Added support for Opti ID OIDC provider
- [ ] Added support for EPiServer OpenIDConnect OIDC provider
- [x] Added support for Azure AD OIDC provider
- [ ] Updated documentation to reflect changes
